src/components/DeckMechanisms/index.js

```diff
@@ -343,6 +385,22 @@ export default class DeckMechanisms extends React.Component {
         break
       }
 
+      // Spellbinder Zhevana
+      case 'W8': {
+        if (this.props.mode === 'AUTOMATIC') {
+          this.setState(state => ({
+            mana: state.mana + state.specifics.frozenEnemiesLevel * 4,
+            specifics: {
+              ...state.specifics,
+              frozenEnemiesLevel: parseInt(
```

Comment:
Why do you need the `parseInt` here?

Reply:
I wanted the Level to be an integer and thought that 3/2 would give 1.5

Reply:
Oh, I see. Maybe `Math.floor` would be more explicit?

Reply:
Yes, that's way better ;)